server/admin_management/api/app/constants.py

```diff
@@ -12,4 +12,17 @@ class UserType(str, Enum):
     BCEID = "B"
 
 
+class AdminRoleAuthGroup(str, Enum):
+    """
+    FAM data model does not explicitly have these role group of admins.
+    However, business rules do differentiate purpose of admins as:
+        (FAM_ADMIN, [APP]_ADMIN, DELEGATED_ADMIN)
+    # Referencing to FAM confluence for design:
+      https://apps.nrs.gov.bc.ca/int/confluence/display/FSAST1/Delegated+Access+Administration+Design (Auth Function)
+    """
+    FAM_ADMIN = "FAM_ADMIN"
+    APP_ADMIN = "APP_ADMIN"
+    DELEGATED_ADMIN = "DELEGATED_ADMIN"
+
```

Comment:
I promote **"FAM_ADMIN"** to a enum to be used for code and tests.
Also added "APP_ADMIN" and "DELEGATED_ADMIN" enum values since soon based on design we could prepare for these constants.